README.md

```diff
@@ -45,6 +44,15 @@ If you are running on Windows we recommend using [Docker and WSL2](https://docs.
 
 There are many projects that can be built and run. [Click here to see the full list](https://github.com/island-is/island.is/blob/main/nx.json).
 
+### Fresh start
+
+Install dependencies and generate some library files:
+
+```bash
+yarn install
+yarn schemas
+```
+
```

Comment:
Devs often need to generate schemas when they switch between branches with different schemas or pull latest from main.

Devs might be used to `yarn install` also generating schemas, so the docs should explain that devs now need to also run `yarn schemas` manually.